core/src/main/java/org/apache/iceberg/ManifestLists.java

```diff
@@ -47,26 +50,52 @@ static List<ManifestFile> read(InputFile manifestList) {
     }
   }
 
+  // or should we modify all related tests (to pass PlaintextEncryptionManager)?
```

Comment:
5 calls in `iceberg-core.test`, 2 calls in `iceberg-core.jmh`

Reply:
To close this out, shall we pick one and remove the inline comment? I prefer passing `PlaintextEncryptionManager`, sine that's the API we want to adopt going forward.

Reply:
> I prefer passing `PlaintextEncryptionManager`, sine that's the API we want to adopt going forward.

To clarify: the preference is to remove this extra API (in ManifestLists.java), and pass `PlaintextEncryptionManager` to the 7 classes in `iceberg-core.test` and `iceberg-core.jmh` ?
(I'm fine either way)

Reply:
SGTM